src/eip7594/poly.c

```diff
@@ -31,7 +31,7 @@
  * Multiplies each coefficient by `shift_factor ^ i`. Equivalent to creating a polynomial that
  * evaluates at `x * shift_factor` rather than `x`.
  *
- * @param[in,out]   p               The polynomial coefficients to be scaled
+ * @param[in,out]   p               The polynomial coefficients to be scaled, length `p`
```

Comment:
Should it be the following?
```suggestion
 * @param[in,out]   p               The polynomial coefficients to be scaled, length `len`
```

Reply:
Yes, it should... good catch again.